index.js

```diff
@@ -10,9 +10,8 @@ const request = require('request');
 
 // Internal Modules
 const connectionManager = require('./lib/connection-manager');
-const distro = require('./lib/os-version');
+const getDistro = require('./lib/getDistro');
```

Comment:
So you probably want to re-test - this doesn't work. You renamed `getDistro.js`.

Reply:
Done, @dchai76!